File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Estatus.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Estatus implements Serializable {
    private int idestatus;
    private String descripcion;
    private String estado;
    private int id_base;

    public Estatus() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Estatus(int idestatus, String descripcion, String estado) {
        this.idestatus = idestatus;
        this.descripcion = descripcion;
        this.estado = estado;
    }

    public int getIdestatus() {
        return idestatus;
    }

    public void setIdestatus(int idestatus) {
        this.idestatus = idestatus;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }
}

File: webservices/CORE/Estatus.php
<?php


class Estatus
{
    public $idestatus;
    public $nombre;
    public $estado;

    /**
     * Estatus constructor.
     * @param $idestatus
     * @param $descripcion
     * @param $estado
     */
    public function __construct($idestatus, $descripcion, $estado)
    {
        $this->idestatus = $idestatus;
        $this->nombre = $descripcion;
        $this->estado = $estado;
    }

    /**
     * @return mixed
     */
    public function getIdestatus()
    {
        return $this->idestatus;
    }

    /**
     * @param mixed $idestatus
     */
    public function setIdestatus($idestatus)
    {
        $this->idestatus = $idestatus;
    }

    /**
     * @return mixed
     */
    public function getDescripcion()
    {
        return $this->nombre;
    }

    /**
     * @param mixed $descripcion
     */
    public function setDescripcion($descripcion)
    {
        $this->nombre = $descripcion;
    }

    /**
     * @return mixed
     */
    public function getEstado()
    {
        return $this->estado;
    }

    /**
     * @param mixed $estado
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    }


}